DockerHubRepo.json

```diff
@@ -0,0 +1,6 @@
+{
+    "repository": {
+        "description": "Description from JSON file!",
+        "full_description": "Full description from JSON file!!!"
```

Comment:
Is the intent to update this before merging?

Reply:
Yes, that was a first thought, but I'm thinking about just updating the full_description with the contents of README.md to avoid having to change multiple files at every release. But that would demand structuring the README.md for that purpose.
WDYT?

Reply:
TBH my feeling here is that simply entering a decent description in the repo metadata manually should be sufficient, new uploads will automatically be reflected in the tags tab when they are pushed (see https://hub.docker.com/r/library/docker/tags/ for example) and the repo description should only change relatively rarely.

I don't know what syntax would be required of `README.md` for it to be suitable for use here, nor whether it would be compatible with the github rendering of the README.